index.html

```diff
@@ -170,6 +170,30 @@ <h3>skills</h3>
                     
                 </section>
         </article>
+        <footer>
+            <div class="data">
+            <p class="comment">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+            <form  action="https://formspree.io/f/xqkovozr" method="POST" class="form">
+              <input class="input_name" type="text" placeholder="Full name" name="name" maxlength="30" required/>
+              <input class="input_email" type="email" placeholder="Email address" name="email" required="email"/>
```

Comment:
- [ ] Kindly make sure your attributes are typed in correctly. The `required` doesn't suppose to have any value as you may notice you literally set it to "email'. Please make sure you delete that value.
```suggestion
              <input class="input_email" type="email" placeholder="Email address" name="email" required />
```

Reply:
Thank you